xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -984,6 +992,30 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
         }
     }
     
+    /**
+     * Run the custom build command specified in "build" parameter
+     */
+    protected def runBuildCommand() {
+	var _buildCommand = newArrayList('-c', targetBuildCommand)
+        val buildCommand = createCommand("bash", _buildCommand)
+	
+        if (buildCommand === null) {
+            return
```

Comment:
Maybe we should throw a warning or an error here.

Reply:
I agree.